Islandora-CLAW/islandora
# Add controlled access terms feature

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/902

* Related PR: https://github.com/Islandora-CLAW/controlled_access_terms/pull/6

# What does this Pull Request do?

Adds a new feature with the configuration for controlled_access_terms.

# What's new?
* Does this change require documentation to be updated? 
   Yes, this feature could use some documentation.
* Does this change add any new dependencies? 
   This feature depends on the controlled_access_terms module, which has its own dependencies.
* Does this change require any other modifications to be made to the repository
 (ie. Regeneration activity, etc.)?  no
* Could this change impact execution of existing code? no

# How should this be tested?

Install controlled_access_terms at the related PR (above) and then install this feature and see if all the fields end up configured as expected.

# Interested parties
@seth-shaw-unlv  @Islandora-CLAW/committers
